addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -423,11 +471,31 @@ options.registry.WebsiteFormEditor = FormEditor.extend({
      * ie: The Job you apply for if the form is on that job's page.
      */
     addActionField: function (previewMode, value, params) {
+        // Remove old property fields.
+        authorizedFieldsCache.get(this.$target[0], this._rpc.bind(this)).then((fields) => {
+            for (const [fieldName, field] of Object.entries(fields)) {
+                if (field._property) {
+                    for (const inputEl of this.$target[0].querySelectorAll(`[name="${fieldName}"]`)) {
```

Comment:
IMO no: `forEach` involves an additional function call at each iteration, and a call of `forEach` itself, and therefore push and pop of parameters etc on the stack. Not a valid argument though given how negligible it is in this context anyway.
Source-wise, it could make sense to have everything on a single line, but in this case, it would lead to a rather long expression:
```js
this.$target[0].querySelectorAll(`[name="${fieldName}"]`).forEach((inputEl) => inputEl.closest(".s_website_form_field").remove());
```
Also, I think it will be easier to maintain if it remains on distinct lines, separating the loop concern from the operation block - in which case we do not need `forEach`.
Bonus: given that it gives no additional feature compared to the language `for` construct, it can be misleading and make devs believe it has an actual purpose. (E.g. there are a few `await stuff.forEach(...)` in the codebase)
(All in bad faith :stuck_out_tongue: )